test/jdk/java/awt/Checkbox/CheckboxCheckerScalingTest.java

```diff
@@ -46,40 +46,47 @@
 public class CheckboxCheckerScalingTest {
     private static Frame frame;
     private static Checkbox checkbox;
-    private static BufferedImage imageAfterChecked;
+    private static volatile Point point;
     private static volatile boolean checkmarkFound = false;
+    private static final int TOLERANCE = 5;
+    private static final int COLOR_CHECK_THRESHOLD = 8;
+    private static int colorCounter = 0;
 
     public static void main(String[] args) throws Exception {
         System.setProperty("sun.java2d.uiScale", "2");
         Robot robot = new Robot();
+
         try {
             EventQueue.invokeAndWait(() -> {
                 frame = new Frame("ComboBox checker scaling test");
```

Comment:
I just noticed the title contains `ComboBox checker`. Should it be `CheckBox checker` ?